rootfs/deis/production.py

```diff
@@ -1,6 +1,4 @@
-import os
```

Comment:
The calls would have to be changed to `--settings deis.production_settings` or we can do `deis.settings.production` if we make it a module

If you look at things in `production.py` tho then you see they are all secrets at this point and in k8s 1.2 we can mount them as ENV vars and it can all move into `settings.py` proper, killing off `production.py`